src/features/auth/lib/authMachine.ts

```diff
@@ -107,6 +127,33 @@ export const authMachine = createMachine<
       },
     },
 
+    creating: {
+      invoke: {
+        src: async (context, event) => {
+          const charityAddress = (event as CreateFarmEvent)
+            .charityAddress as CharityAddress;
+          const donation = (event as CreateFarmEvent).donation as number;
+
+          await createFarm(charityAddress, donation);
+        },
+        onDone: "loadingFarms",
+        onError: {
+          target: "unauthorised",
+          actions: assign({
```

Comment:
<img width="563" alt="Screen Shot 2022-02-01 at 10 31 08 am" src="https://user-images.githubusercontent.com/11745561/151977424-1e0bf3cc-961a-4435-838b-fc8c46a16215.png">

When the API is not running, the `createFarm` errors out. For some reason it does not transition into the `unauthorised` state 🤔